app/models/miq_expression.rb

```diff
@@ -371,56 +371,56 @@ def to_human
     self.class._to_human(@exp)
   end
 
-  def self._to_human(exp, options = {})
-    return exp unless exp.kind_of?(Hash) || exp.kind_of?(Array)
+  def self._to_human(expr, options = {})
```

Comment:
I don't think these are a problem either - these `exp`s are at a different scope to the instance-level `exp` getter, and will never be mistaken